ngraph_bridge/ngraph_executor.cc

```diff
@@ -477,32 +477,30 @@ NGraphExecutor::InitializeIOTensorPipeline(
         nullptr);
   }
   // Create these pipelined ng tensors only if needed, else reuse from cache
-  size_t num_pipelined_inputs = pipelined_input_indexes.size();
-  size_t num_pipelined_outputs = pipelined_output_indexes.size();
+  size_t num_inputs = ng_exec->get_parameters().size();
```

Comment:
this seems incorrect. please take the incoming change
size_t num_pipelined_inputs = pipelined_input_indexes.size();
  size_t num_pipelined_outputs = pipelined_output_indexes.size();